apps/docs/pages/guides/database/postgres/row-level-security.mdx

```diff
@@ -105,12 +105,12 @@ alter table profiles enable row level security;
 create policy "Users can create a profile."
 on profiles for insert
 to authenticated                    -- the Postgres Role (recommended)
-using ( auth.id() = user_id );      -- the actual Policy
+with check ( auth.id() = user_id );      -- the actual Policy
```

Comment:
I'm not sure that `auth.id()` is actually valid here

I ran into this issue while walking through the docs. Should be `auth.uid()` instead

Reply:
Done.